xt/66-cucumber/01-basic/setup.feature

```diff
@@ -171,9 +171,9 @@ Scenario: Create database with ampersand in the name
       | First Name         | A                |
       | Last name          | Dmin             |
       | Employee Number    | 00000001         |
+      | Country            | United States    |
       | Date of Birth      | 09/01/2006       |
       | Tax ID/SSN         | 00000002         |
-      | Country            | United States    |
```

Comment:
Can we create a test with a localized version as well? Maybe a French or German version?

That'll guarantee our support stays in place.